app/views/listings/_listing_actions.haml

```diff
@@ -69,7 +69,7 @@
         .content
           = @listing.transaction_type.action_button_label(I18n.locale)
 
-    - if payment_gateway == :paypal
+    - if payment_gateway == :paypal && @listing.transaction_type.status_after_reply != "free"
```

Comment:
It seems that `status_after_reply` takes price into account, so my previous comment is not true.